src/_guides/language/language-tour.md

```diff
@@ -2891,32 +2921,89 @@ class SmartTelevision extends Television {
 }
 {% endprettify %}
 
-Subclasses can override instance methods, getters, and setters. Here’s
-an example of overriding the Object class’s `noSuchMethod()` method,
-which is called whenever code attempts to use a non-existent method or
-instance variable:
 
-<!-- language-tour/no-such-method/bin/main.dart -->
+#### Overriding members
+
+Subclasses can override instance methods, getters, and setters.
+You can use the `@override` annotation to indicate that you are
+intentionally overriding a member:
+
+<!-- language-tour/overrides/bin/override.dart -->
 {% prettify dart %}
-class A {
-  // Unless you override noSuchMethod, using a
-  // non-existent member results in a NoSuchMethodError.
-  void noSuchMethod(Invocation mirror) {
-    print('You tried to use a non-existent member:' +
-          '${mirror.memberName}');
+class SmartTelevision extends Television {
+  @override
+  void turnOn() {
+    // ...
   }
+  // ...
 }
 {% endprettify %}
 
-You can use the `@override` annotation to indicate that you are
-intentionally overriding a member:
+To narrow the type of a method parameter while maintaining type safety,
+use the `covariant` keyword.
```

Comment:
As I mentioned in the email message, `covariant` is useless in Dart 1. This document never says 'strong mode' anywhere, so maybe we shouldn't mention `covariant` here at all?

Otherwise, it should be described in a separate section about strong mode, and the only reference to `covariant` in the Dart 1 part should be `You can add the modifier `covariant` to parameter declarations in instance methods (including setters and operators); it has no effect. It is only allowed because it can make the same code usable in Dart 1 as well as in strong mode.'.